.travis.yml

```diff
@@ -64,3 +64,4 @@ after_failure:
   - sudo cat /var/log/php-fpm.log
   - sudo ls -l /var/log/apache2
   - sudo cat /var/log/apache2/other_vhosts_access.log
+  - find . -name "*.php" ! -path "./vendor/*" ! -path "./tools/*" -print0 | xargs -0 -n1 -P4 php -l
```

Comment:
One more syntax check?

Reply:
nope, I've moved the check here in order to see which php files are corrupted

Reply:
I probably miss something, did you remove a check on another line?

Checking for syntax errors only in case a failure is a bad idea, as we may miss something if all the tests are good.

Reply:
not moved: added

Reply:
To be removed in this PR.
If we need to display the files with a syntax error, it must be done properly in the original script.